include/Encode/AlphaEncodingManager.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include <Encode/EncodingManager.h>
+
+class AlphaEncodingManager : public IEncodingManager {
+public:
+	AlphaEncodingManager(float maxAnalogValue) : m_maxAnalogValue(maxAnalogValue){};
+	
+	//decode the given string into a VRCommData_t
+	VRCommData_t Decode(std::string input);
+private:
+    std::string getArgumentSubstring(std::string str, char del);
+
+	float m_maxAnalogValue;
+	const char* alphabet = "ABCDE";  // expand as more letters are added to manager
```

Comment:
is this variable used? When decoding it looks like you have letters after `E` as well

Reply:
Great catch, thank you. Wasn't testing with a joystick so I did not notice